moveit_core/collision_detection_fcl/src/collision_robot_fcl.cpp

```diff
@@ -41,6 +41,8 @@
 #include <fcl/broadphase/broadphase_dynamic_AABB_tree.h>
 #endif
 
+rclcpp::Logger LOGGER_COLLISION_ROBOT_FCL = rclcpp::get_logger("collision_robot.fcl");
```

Comment:
move inside the namespace

Reply:
https://github.com/ros-planning/moveit2/pull/41/commits/a3011f83f7a76ca3856b5e8b490f587b1af43b7d